app/models/deferred/visitor.rb

```diff
@@ -0,0 +1,4 @@
+module Deferred
+  class Visitor < ::Visitor
+  end
```

Comment:
My thinking was more to make it explicit to anyone new coming to that particular class that WRITE THEE NO METHODS HERE: 

Something like: 

```
module Deferred 
  class Visitor < ::Visitor 
    
    class WriteTheeNoMethodsHere < Error 
       "This is a DANGEROUS place.  Leave here now..." 
    end

    def initialize 
       raise WriteTheeNoMethodsHere if self.instance_methods(false)
    end
end
```